### PR TITLE
Merged SLE-12-SP1

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :sle12sp2
```
Comment (jreidinger) [inline Rakefile]: this is wrong. there should not be anything, otherwise public jenkins also send it to SP2

Comment (mchf) [inline Rakefile]: Thanks. Obviously I live in SLE world only (master == SLE12SP2) :-/

**Comment by mvidner** [comment]
Travis failed because in the two branches we have refactored the code and the tests independently and Git is not smart enough to merge that.
But I see the last commit corrects that, good.

**Comment by jreidinger** [comment]
LGTM

**Comment by mvidner** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for the reviews.

**Pull request merged**